doc/foreign-code.rst

```diff
@@ -335,26 +334,26 @@ follows:
     (name bar)
     (foreign_archives foo))
 
-Then, whenever you use the ``bar`` library, you will also be able to
+Then, whenever you use the ``bar`` library, you'll also be able to
 use C functions from ``libfoo``.
 
 Limitations
 -----------
 
 When using the sandboxing method, the following limitations apply:
 
-- the build of the foreign code will be sequential
-- the build of the foreign code won't be incremental
+- The build of the foreign code will be sequential
+- The build of the foreign code won't be incremental
 
-both these points could be improved. If you are interested in helping
+Both these points could be improved. If you're interested in helping
 make this happen, please let the Dune team know and someone will guide
 you.
 
-Real example
+Real Example
 ------------
 
 The `re2 project <https://github.com/janestreet/re2>`_ uses this
-method to build the re2 C library. You can look at the file
+method to build the ``re2`` C library. You can look at the file
```

Comment:
Same here. re2 is does not appear in source code so it should be single quoted as in:

```
`re2` C library
```

or just left alone as re2.